<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1613</width>
    <height>910</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTreeWidget" name="treeWidget">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>210</y>
      <width>971</width>
      <height>401</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>成员名称</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>成员信息</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>配偶信息</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>760</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="findButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>770</y>
      <width>91</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="modifyButton">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>770</y>
      <width>101</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>、</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="parentEdit">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>660</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameEdit">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>660</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="detailsEdit">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>660</y>
      <width>401</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="createButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>770</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="switchButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>770</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="familyNameEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>620</y>
      <width>491</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QListWidget" name="familyTreeList">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>511</width>
      <height>391</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addSpouseButton">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>770</y>
      <width>91</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>1600</width>
      <height>900</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/image/image/background.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="addSiblingButton">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>760</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="modifySpouseButton">
    <property name="geometry">
     <rect>
      <x>1340</x>
      <y>760</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="exportButton">
    <property name="geometry">
     <rect>
      <x>1490</x>
      <y>760</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>switchButton</zorder>
   <zorder>addButton</zorder>
   <zorder>addSpouseButton</zorder>
   <zorder>findButton</zorder>
   <zorder>modifyButton</zorder>
   <zorder>createButton</zorder>
   <zorder>familyTreeList</zorder>
   <zorder>treeWidget</zorder>
   <zorder>parentEdit</zorder>
   <zorder>nameEdit</zorder>
   <zorder>detailsEdit</zorder>
   <zorder>familyNameEdit</zorder>
   <zorder>addSiblingButton</zorder>
   <zorder>modifySpouseButton</zorder>
   <zorder>exportButton</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1613</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
